tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/validation/TclMissingFixtureValidatorTest.xtend

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.tcl.dsl.validation
+
+import org.eclipse.xtext.common.types.JvmOperation
+import org.eclipse.xtext.common.types.JvmParameterizedTypeReference
+import org.eclipse.xtext.common.types.JvmType
+import org.eclipse.xtext.validation.ValidationMessageAcceptor
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentCaptor
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+import org.testeditor.aml.AmlFactory
+import org.testeditor.aml.InteractionType
+import org.testeditor.aml.MethodReference
+import org.testeditor.aml.dsl.AmlStandaloneSetup
+import org.testeditor.aml.impl.AmlFactoryImpl
+import org.testeditor.tcl.dsl.tests.parser.AbstractParserTest
+import org.testeditor.tcl.util.TclModelUtil
+
+import static org.mockito.Matchers.*
+
+import static extension org.mockito.Mockito.*
+
+class TclMissingFixtureValidatorTest extends AbstractParserTest {
+
+	@Mock TclModelUtil tclModelUtil // injected into class under test
+	@InjectMocks TclValidator tclValidator // class under test
+	@Mock JvmParameterizedTypeReference typeReferenceMock
+	@Mock ValidationMessageAcceptor messageAcceptor
+
+	var AmlFactory amlFactory
```

Comment:
I would either use the default EMF mechanism here (`static val amlFactory = AmlFactory.eInstance`) or bind the `AmlFactory` to `AmlFactoryImpl` in the aml runtime module.